main.py

```diff
@@ -1,10 +1,53 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def get_race(race_info: dict) -> Race:
+    race, _ = Race.objects.get_or_create(
+        name=race_info["name"],
+        description=race_info["description"]
+    )
+    for skill_info in race_info["skills"]:
+        _ = get_skill(skill_info, race)
```

Comment:
Why you create a var here?

Reply:
@Oleg-s-s Discards are placeholder variables that are intentionally unused in application code. Discards make the intent of your code clear. A discard indicates that our code never uses the variable. They enhance its readability and maintainability.